### ISSUE TITLE
add support for Eigen::Array

### ISSUE THREAD

**Issue by xzz53** [BlueBrain/HighFive#365]
It would be very convenient to allow `DataSpace::From` to support `Eigen::Array`s the same way it supports `Eigen::Matrix`'s.

**Comment by alkino** [comment]
We will fix this after #307

**Comment by alkino** [comment]
It should be fixed now, in fact?

`DataSpace::From` has been rewrite. Can you test please?

**Comment by xzz53** [comment]
@alkino , `DataSpace::From` really works with `Eigen::Array` now, thanks!

Unfortunately, it looks like `DataSet::write` still doesn't support `Eigen::Array`'s, although `H5Easy::dump` does. Should I create a separate issue for that?

**Comment by alkino** [comment]
no this is in the previous ticket, sorry.
